src/net/sourceforge/kolmafia/persistence/HolidayDatabase.java

```diff
@@ -820,65 +727,94 @@ public String toString() {
     }
   }
 
-  public static String getHoliday() {
-    return getHoliday(false);
+  public static String getHolidayInDays(final int days) {
+    return getHolidayInDays(DateTimeManager.getRolloverDateTime(), days);
   }
 
-  public static String getHoliday(final boolean showPredictions) {
-    return getHoliday(DateTimeManager.getRolloverDateTime(), showPredictions);
+  public static String getHolidayInDays(final ZonedDateTime dateTime, final int days) {
+    int calendarDay = getDayInKoLYear(dateTime);
+    return getGameHoliday((calendarDay + days) % 96);
   }
 
-  public static String getHoliday(final ZonedDateTime dateTime) {
-    return getHoliday(dateTime, false);
+  public static void setHoliday(final String holiday) {
+    HOLIDAY_OVERRIDE = holiday.isBlank() ? null : holiday;
   }
 
   /** Returns the KoL holiday associated with the given date in the real world. */
-  public static String getHoliday(final ZonedDateTime dateTime, final boolean showPrediction) {
+  public static List<String> getHolidays(final ZonedDateTime dateTime) {
+    var holidays = new ArrayList<String>();
+
     String gameHoliday = getGameHoliday(dateTime);
+    if (gameHoliday != null) holidays.add(gameHoliday);
+
     String realHoliday = getRealLifeHoliday(dateTime);
+    if (realHoliday != null) holidays.add(realHoliday);
 
-    if (showPrediction && realHoliday == null) {
-      if (gameHoliday != null) {
-        return gameHoliday + " today";
-      }
+    if (holidays.contains("St. Sneaky Pete's Day") && holidays.contains("Feast of Boris")) {
+      holidays.clear();
+      holidays.add("Drunksgiving");
```

Comment:
if you care about Drunksgiving may as well care about El Dia de los Muertos Borrachos y Agradecido. Or was Drunksgiving there before?

Reply:
It was there before